homeassistant/components/switchbot/quality_scale.yaml

```diff
@@ -0,0 +1,66 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: todo
+  common-modules: done
+  config-flow-test-coverage: todo
```

Comment:
Only patch the library where you use them

```py
    with patch(
        "switchbot.SwitchbotLock.verify_encryption_key",
        return_value=False,
    ):
```
so rather do `homeassistant.components.switchbot.....`